### PR TITLE
N°5009 - Invalid XML for class Attachment (icon node)

### PR DESCRIPTION
**Pull request by eespie**
Realign all XML files  into datamodels/2.X to match the format 3.0

 * moved class/properties/icon to class/properties/style/icon

This allows previous redefinition of icons (they failed in 3.0-3.0.1)

### CODE REVIEW

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 1]
```diff
@@ -362,7 +362,9 @@ public function PrefillSearchForm(&$aContextParam)
 						<attribute id="name"/>
 					</attributes>
 				</naming>
-				<icon/>
+        <style>
+          <icon/>
+        </style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]: Mind the indentation. Unfortunately some XML files use tabs, others use spaces 😕
Until we harmonize it all it's best to use the same indent type as the rest of the file to ease readability.

_Note: I couldn't provide a suggestion with tab characters on the GitHub UI_

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 2]
```diff
@@ -737,7 +739,9 @@ public function PrefillSearchForm(&$aContextParam)
 						<attribute id="contact_id"/>
 					</attributes>
 				</naming>
-				<icon/>
+        <style>
+          <icon/>
+        </style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]: Same comment on indentation

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 3]
```diff
@@ -817,7 +821,9 @@ public function PrefillSearchForm(&$aContextParam)
 						<attribute id="document_id"/>
 					</attributes>
 				</naming>
-				<icon/>
+        <style>
+          <icon/>
+        </style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]: Same comment on indentation

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 4]
```diff
@@ -1207,7 +1213,9 @@ public function PrefillSearchForm(&$aContextParam)
...
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]: Same comment on indentation

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 5]
```diff
@@ -1287,7 +1295,9 @@ public function PrefillSearchForm(&$aContextParam)
 						<attribute id="contact_id"/>
 					</attributes>
 				</naming>
-				<icon/>
+        <style>
+          <icon/>
+        </style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]: Same comment on indentation

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 6]
```diff
@@ -1842,7 +1852,9 @@ public function PrefillSearchForm(&$aContextParam)
 						<attribute id="slt_id"/>
 					</attributes>
 				</naming>
-				<icon/>
+        <style>
+          <icon/>
+        </style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]: Same comment on indentation

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 7]
```diff
@@ -1962,7 +1974,9 @@ public function PrefillSearchForm(&$aContextParam)
 						<attribute id="service_id"/>
 					</attributes>
 				</naming>
-				<icon/>
+        <style>
+          <icon/>
+        </style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]: Same comment on indentation

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 8]
```diff
@@ -2058,7 +2072,9 @@ public function PrefillSearchForm(&$aContextParam)
 						<attribute id="providercontract_id"/>
 					</attributes>
 				</naming>
-				<icon/>
+        <style>
+          <icon/>
+        </style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]: Same comment on indentation

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 9]
```diff
@@ -2243,7 +2259,9 @@ public function PrefillSearchForm(&$aContextParam)
 						<attribute id="contact_id"/>
 					</attributes>
 				</naming>
-				<icon/>
+        <style>
+          <icon/>
+        </style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]: Same comment on indentation

File: datamodels/2.x/itop-tickets/datamodel.itop-tickets.xml [hunk 10]
```diff
@@ -27,7 +27,9 @@
 						<attribute id="ref"/>
 					</attributes>
 				</naming>
-				<icon/>
+        <style>
+          <icon/>
+        </style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-tickets/datamodel.itop-tickets.xml]: Same comment on indentation

File: datamodels/2.x/itop-tickets/datamodel.itop-tickets.xml [hunk 11]
```diff
@@ -393,7 +395,9 @@
 						<attribute id="contact_id"/>
 					</attributes>
 				</naming>
-				<icon/>
+        <style>
+          <icon/>
+        </style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-tickets/datamodel.itop-tickets.xml]: Same comment on indentation

**Review by Molkobain — CHANGES_REQUESTED** [review]
Apart from the indentation comments it seems all good.

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml (hunk 1 above)
Comment (Hipska) [reply datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]: Hint; you can type a tab character in another application and copy it in here..

Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
		<style>
			<icon/>
		</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml (hunk 2 above)
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
		<style>
			<icon/>
		</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml (hunk 3 above)
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
		<style>
			<icon/>
		</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml (hunk 4 above)
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
		<style>
			<icon/>
		</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml (hunk 5 above)
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
		<style>
			<icon/>
		</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml (hunk 6 above)
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
		<style>
			<icon/>
		</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml (hunk 7 above)
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
		<style>
			<icon/>
		</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml (hunk 8 above)
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
		<style>
			<icon/>
		</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml (hunk 9 above)
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
		<style>
			<icon/>
		</style>
```

File: datamodels/2.x/itop-tickets/datamodel.itop-tickets.xml (hunk 10 above)
Comment (Molkobain) [inline datamodels/2.x/itop-tickets/datamodel.itop-tickets.xml]:
```suggestion
		<style>
			<icon/>
		</style>
```

File: datamodels/2.x/itop-tickets/datamodel.itop-tickets.xml (hunk 11 above)
Comment (Molkobain) [inline datamodels/2.x/itop-tickets/datamodel.itop-tickets.xml]:
```suggestion
		<style>
			<icon/>
		</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 12]
```diff
@@ -362,7 +362,9 @@ public function PrefillSearchForm(&$aContextParam)
 						<attribute id="name"/>
 					</attributes>
 				</naming>
-				<icon/>
+		<style>
+			<icon/>
+		</style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
				<style>
					<icon/>
				</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 13]
```diff
@@ -737,7 +739,9 @@ public function PrefillSearchForm(&$aContextParam)
 						<attribute id="contact_id"/>
 					</attributes>
 				</naming>
-				<icon/>
+		<style>
+			<icon/>
+		</style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
				<style>
					<icon/>
				</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 14]
```diff
@@ -817,7 +821,9 @@ public function PrefillSearchForm(&$aContextParam)
 						<attribute id="document_id"/>
 					</attributes>
 				</naming>
-				<icon/>
+		<style>
+			<icon/>
+		</style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
				<style>
					<icon/>
				</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 15]
```diff
@@ -1207,7 +1213,9 @@ public function PrefillSearchForm(&$aContextParam)
...
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
				<style>
					<icon/>
				</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 16]
```diff
@@ -1287,7 +1295,9 @@ public function PrefillSearchForm(&$aContextParam)
 						<attribute id="contact_id"/>
 					</attributes>
 				</naming>
-				<icon/>
+		<style>
+			<icon/>
+		</style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
				<style>
					<icon/>
				</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 17]
```diff
@@ -1842,7 +1852,9 @@ public function PrefillSearchForm(&$aContextParam)
 						<attribute id="slt_id"/>
 					</attributes>
 				</naming>
-				<icon/>
+		<style>
+			<icon/>
+		</style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
				<style>
					<icon/>
				</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 18]
```diff
@@ -1962,7 +1974,9 @@ public function PrefillSearchForm(&$aContextParam)
 						<attribute id="service_id"/>
 					</attributes>
 				</naming>
-				<icon/>
+		<style>
+			<icon/>
+		</style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
				<style>
					<icon/>
				</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 19]
```diff
@@ -2058,7 +2072,9 @@ public function PrefillSearchForm(&$aContextParam)
 						<attribute id="providercontract_id"/>
 					</attributes>
 				</naming>
-				<icon/>
+		<style>
+			<icon/>
+		</style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
				<style>
					<icon/>
				</style>
```

File: datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml [hunk 20]
```diff
@@ -2243,7 +2259,9 @@ public function PrefillSearchForm(&$aContextParam)
 						<attribute id="contact_id"/>
 					</attributes>
 				</naming>
-				<icon/>
+		<style>
+			<icon/>
+		</style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-service-mgmt/datamodel.itop-service-mgmt.xml]:
```suggestion
				<style>
					<icon/>
				</style>
```

File: datamodels/2.x/itop-tickets/datamodel.itop-tickets.xml [hunk 21]
```diff
@@ -27,7 +27,9 @@
 						<attribute id="ref"/>
 					</attributes>
 				</naming>
-				<icon/>
+		<style>
+			<icon/>
+		</style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-tickets/datamodel.itop-tickets.xml]:
```suggestion
				<style>
					<icon/>
				</style>
```

File: datamodels/2.x/itop-tickets/datamodel.itop-tickets.xml [hunk 22]
```diff
@@ -393,7 +395,9 @@
 						<attribute id="contact_id"/>
 					</attributes>
 				</naming>
-				<icon/>
+		<style>
+			<icon/>
+		</style>
```
Comment (Molkobain) [inline datamodels/2.x/itop-tickets/datamodel.itop-tickets.xml]:
```suggestion
				<style>
					<icon/>
				</style>
```

**Review by Molkobain — APPROVED** [review]

**Pull request merged**